notanorm/mysql.py

```diff
@@ -228,19 +236,24 @@ def simplify_model(model: DbModel):
         return model2
 
     def column_model(self, info, in_primary):
-        # depends on specific mysql version, these are display width hints
-
-        if info.type == "int(11)" or info.type == "int":  # pragma: no cover
+        # depends on specific mysql version, these are often display width hints
```

Comment:
Out of scope for this, but do we test against different MySQL versions in CI? Or just whatever the default in `ankane-setup-mysql` is? If not, we should probably add.

Reply:
i just have one version on my machine and "whatever" the default is in that setup.   yeah, we should.  could get covrg on these things